www/src/components/guidelines/avatar.js

```diff
@@ -1,16 +1,18 @@
-import styled from "@emotion/styled"
+/** @jsx jsx */
+import { jsx } from "theme-ui"
 
-import { Box } from "./system"
-
-const Avatar = styled(Box)()
-
-Avatar.defaultProps = {
-  bg: `grey.10`,
-  borderRadius: 6,
-  flex: `0 0 auto`,
-  height: `avatar`,
-  lineHeight: `solid`,
-  width: `avatar`,
-}
+const Avatar = ({ ...rest }) => (
+  <div
+    sx={{
+      bg: `grey.10`,
+      borderRadius: 6,
+      flex: `0 0 auto`,
+      height: `avatar`,
+      lineHeight: `solid`,
+      width: `avatar`,
+    }}
+    {…rest}
```

Comment:
```suggestion
    {...rest}
```

Reply:
Why should we having rest there and not on sx, because Avatar is pure UI component and we will only be changing the styles using props and nothing else

Reply:
I think the rest should be inside the SX only

Reply:
<img width="512" alt="avatar" src="https://user-images.githubusercontent.com/19193724/84338542-fd963300-abb9-11ea-8cff-78b77113540e.png">